include/implGBL/EUTelNav.h

```diff
@@ -0,0 +1,93 @@
+/** Basic navigation tools for general track fitting.  
```

Comment:
It tells you:  +/** Basic navigation tools for general track fitting.  - this is not GBL specific!